modules/mobile/app/controllers/mobile/v0/facilities_info_controller.rb

```diff
@@ -20,7 +20,8 @@ def index
       end
 
       def schedulable
-        facility_ids = @current_user.va_treatment_facility_ids + @current_user.cerner_facility_ids
+        facility_ids = (@current_user.va_treatment_facility_ids + @current_user.cerner_facility_ids).uniq
```

Comment:
I believe the old specs just don't have any duplicates but the new specs do and revealed this issue.